[DOC] Update documentation for CopperApps (#14)
Coppernic/coppernic.github.io: copperapps.md

CopperApps
=======

![](_media/copperapps.png)

> We are using a custom version of [CopperApps](https://CopperApps.org/en/) application to deliver our samples and other apps. This application is named **CopperApps**

Reviewer: Quel est ce lient CopperApps.org? C'est pas PLutot [F-Droid] ( fdroid.org )?

Reviewer 2: ```suggestion
We are using [CopperApps](https://CopperApps.org/en/) application to deliver our samples, services and other apps like tools.
```

Download
--------

> CopperApps build by Coppernic is available [here](https://github.com/Coppernic/coppernic.github.io/releases/tag/1.8.1).

Reviewer: pas de lien (facile) a taper sur son navigateur comme www.coppernic.fr/copperapps.apk ?

Reviewer 2: ```suggestion
CopperApps build by Coppernic is available on Github [here](https://github.com/Coppernic/coppernic.github.io/releases/tag/1.8.1) or directly from the web browser of your PDA as an APK through coppernic.fr/copperaps.apk.
```

> To install it on a device, please click on [this link](https://github.com/Coppernic/coppernic.github.io/releases/download/1.8.1/CopperApps-full-coppernic-external-standard-1.8.1.apk) from Android, download will start and then you will be able to install it on your device.

Reviewer 2: Cela semble redondant avec la ligne du dessus (11). Faudrait fusionner ;-)

Author: Ce n'est pas tout à fait les mêmes liens

It embeds Coppernic's repository:

- Common repo

>     - `http://fdroid.coppernic.fr/common/fdroid/repo`

Reviewer 2: Faudrait renommer ce repo si possible par une URL CopperApps.

Author: Ce n'est pas d'actualité. D'autant plus qu'on ne cache pas le fait que c'est F-Droid qui est derrière CopperApps.
